packages/backend/src/api.ts

```diff
@@ -36,6 +42,7 @@ export function createAPIApp(): express.Application {
     next();
   });
   app.use(traceAPICall);
+  app.use(validateRequest(routeValidator));
```

Comment:
I think this is fine for now! The main reason I made `authenticateRequest` a function wrapper was that it needed to _output_ the authenticated user ID, and I couldn't figure out a typesafe way to do that with Express middleware normally. (But your question reflects a good instinct, I think)